python/pyproject.toml

```diff
@@ -0,0 +1,81 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = [
+  "hatchling>=1.27.0",
+  "hatch-fancy-pypi-readme",
+]
+
+[project]
+name = "dotslash"
+dynamic = [
+  "readme",
+  "version",
+]
+authors = [
+  { name = "Ofek Lev", email = "oss@ofek.dev" },
+]
+description = "Command-line tool to facilitate fetching an executable, caching it, and then running it."
+keywords = [
+  "dotslash",
+]
+license = "MIT OR Apache-2.0"
+requires-python = ">=3.10"
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "Natural Language :: English",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+
+[project.urls]
+Homepage = "https://dotslash-cli.com"
+Tracker = "https://github.com/facebook/dotslash/issues"
+Source = "https://github.com/facebook/dotslash"
+
+[tool.hatch.metadata.hooks.custom]
+
+[tool.hatch.metadata.hooks.fancy-pypi-readme]
+content-type = "text/markdown"
+
+[[tool.hatch.metadata.hooks.fancy-pypi-readme.fragments]]
+path = "README.md"
+end-before = "## Building from source"
+
+[[tool.hatch.metadata.hooks.fancy-pypi-readme.fragments]]
+path = "README.md"
+start-at = "## License"
+
+[[tool.hatch.metadata.hooks.fancy-pypi-readme.substitutions]]
+pattern = "(?m)^- \\[Building from source.+$\\s+-"
+replacement = "-"
+
+[tool.hatch.build.targets.sdist.force-include]
+"../LICENSE-MIT" = "LICENSE-MIT"
+"../LICENSE-APACHE" = "LICENSE-APACHE"
+
+[tool.hatch.build.targets.wheel.hooks.custom]
+
+[tool.cibuildwheel]
+enable = ["pypy"]
+test-command = "python -m dotslash -- cache-dir"
+# Use UV for build environment creation and installation of build dependencies
+build-frontend = "build[uv]"
+# Only build on one version of Python since all distributions of a given platform/arch pair are the same
+build = "cp314-*"
+# Disable wheel repair since it's not necessary
+repair-wheel-command = ""
+
+[tool.cibuildwheel.linux]
+environment-pass = ["DOTSLASH_SOURCE", "DOTSLASH_VERSION"]
+
+[tool.black]
+target-version = ["py310"]
+
+[tool.usort]
+first_party_detection = false
+
+[tool.ufmt]
+formatter = "ruff-api"
+sorter = "ruff-api"
```

Comment:
I'll fix this locally before merging internally, but I'm going to re-order these to be alphabetical by section.